mlrun/projects/pipelines.py

```diff
@@ -782,6 +782,21 @@ def run(
                 project_name=project.name,
             )
 
+            # set it relative to project path
+            # as the runner pod will mount and use `load_and_run` which will use the project context
+            # to load the workflow file to.
+            # e.g.
+            # /path/to/project/workflow.py -> ./workflow.py
+            # /path/to/project/subdir/workflow.py -> ./subdir/workflow.py
```

Comment:
if you are removing the code path which includes the subpath it will be like this no?
```suggestion
            # /path/to/project/subdir/workflow.py -> ./workflow.py
```